JS-counting/script.js

```diff
@@ -0,0 +1,35 @@
+console.log("Part 1");
+for (i = 1; i <= 10; i++) {
+    console.log(i);
+}
+
+for (i = 10; i >= 1; i--) {
+    console.log(i);
+}
+
+console.log("Part 2")
+let j = 1;
+while (j <= 10) {
+    console.log(j);
+    j++;
+}
+j = 10;
```

Comment:
Add a new line between blocks of code that do different things

```suggestion
}

j = 10
```